.github/workflows/main.yml

```diff
@@ -262,8 +262,6 @@ jobs:
           body: |
             ## Deadlock API Ingest ${{ needs.build.outputs.version }}
 
-            Automated release with the latest changes from master branch.
-
```

Comment:
Why remove this?

I think it's good to notice, that it was an automatically created release.